app/components/contributor-list/template.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if contributors.isPending}}
+    <div class="ball-pulse">
```

Comment:
I'm currently changing some spinner and I think the required nested `div`s is a little ridiculous. Just thought you should know.